packages/text-sets/package.json

```diff
@@ -13,5 +13,7 @@
   "dependencies": {
```

Comment:
Add this above:

```json
"bin": {
    "render-text-sets": "./src/cli.js"
},
```

This makes using the CLI script easier.

For example in the main `package.json` file we could then add:

```json
"workflow:render-text-sets": "render-text-sets",
```

Perhaps with a better name